src/status_im2/contexts/quo_preview/avatars/account_avatar.cljs

```diff
@@ -36,5 +37,16 @@
                              :emoji               "🍑"
                              :type                :default})]
     (fn []
-      [preview/preview-container {:state state :descriptor descriptor}
-       [quo/account-avatar @state]])))
+      [preview/preview-container
+       {:state                     state
+        :descriptor                descriptor
+        :component-container-style {:align-items     :center
+                                    :justify-content :center}}
+       [quo/account-avatar @state]
+       [quo/button
+        {:type            :grey
+         :container-style {:margin-top 30}
+         :on-press        #(rf/dispatch [:emoji-picker/open
```

Comment:
I wonder should we move away from having reframe events in these preview screens. Imo we should treat preview screens as a standalone application - just something we have in the same codebase/application for 
the convenience of avoiding an additional build and release pipeline for a storybook type app.

Reply:
The app's navigation methods are wrapped in Reframe events for certain reasons and updates in app-db. It would be hard if we wanted to away from it.

Reply:
yeah aside from navigation of course :)
I mean for example uses like this, and another example is the toast preview screen is also doing something similar 👍

Reply:
@J-Son89 - One other reason to have a separate event `:emoji-picker/open` is to add the recent emojis (which are more likely to be stored and pulled from AsyncStorage or equivalent) into the emoji picker in future.

Reply:
sure, the event makes sense. I just meant in the context of using it within preview screens. e.g we can just have an emoji descriptor which offers several emojis to test instead

Reply:
anyway it's just something for thought/ we should bring to discussion at some point. Either way is fine for now 👍